actors/builtin/market/policy.go

```diff
@@ -35,17 +36,16 @@ func dealPricePerEpochBounds(_ abi.PaddedPieceSize, _ abi.ChainEpoch) (min abi.T
 	return abi.NewTokenAmount(0), builtin.TotalFilecoin // PARAM_FINISH
 }
 
-func DealProviderCollateralBounds(pieceSize abi.PaddedPieceSize, verified bool, networkQAPower, baselinePower abi.StoragePower, networkCirculatingSupply abi.TokenAmount) (min abi.TokenAmount, max abi.TokenAmount) {
+func DealProviderCollateralBounds(pieceSize abi.PaddedPieceSize, verified bool, networkRawPower, networkQAPower, baselinePower abi.StoragePower,
+	networkCirculatingSupply abi.TokenAmount, _ runtime.NetworkVersion) (min, max abi.TokenAmount) {
```

Comment:
I preserved the network version plumbing, even though it's unused, to aid using it in a possible future inline upgrade.

Reply:
I don't see the value in keeping that plumbing here.  It's not obvious that past upgrade sites are good predictors for future upgrade sites.  This plumbing is very easy to do lazily.  If we keep it around without actually referencing it we add friction to tracing through the code to find existing upgrade sites.

Reply:
Similar thoughts about the version suffix on constants.  When reading the code looking at specs-actors/v2 gives the context for which protocol epochs these constants apply to.  If we need to go back and refer to previous constants we can always add on suffixes as needed here since we'd be doing an upgrade anyway.  We can also make use of importing constants from previous branches.

Reply:
Ok, I'll remove the plumbing